app/services/reporting/total_user_count_report.rb

```diff
@@ -28,5 +28,21 @@ def total_user_count_emailable_report
     def total_user_count
       User.where('created_at <= ?', report_date).count
     end
+
+    def verified_user_count
+      Profile.where(active: true).where('activated_at <= ?', report_date).count
```

Comment:
reminder to add `Reports::BaseReport.transaction_with_timeout { ... }` here

Reply:
or around the generation of the whole report instead of each DB query

Reply:
Thanks! That had completely slipped my mind.